force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.js

```diff
@@ -21,6 +21,8 @@ import stgSimpleAddressChangeUpdate from "@salesforce/label/c.stgSimpleAddressCh
 import stgTabAfflMappings from "@salesforce/label/c.stgTabAfflMappings";
 import stgAffiliationsSettingsNav from "@salesforce/label/c.stgAffiliationsSettingsNav";
 import afflTypeEnforced from "@salesforce/label/c.afflTypeEnforced";
+//Errors Settings Labels
```

Comment:
@m-kinkade I will suggest we get rid of the comment here for consistency purpose.

Reply:
It looks like there are currently comments denoting the other custom labels for settings, I though including this comment was consistent with what already exists in the file?